s2fft/logs.py

```diff
@@ -29,7 +32,7 @@ def setup_logging(custom_yaml_path: str = None):
         path = Path(custom_yaml_path)
     if not path.exists():
         raise ValueError(f"Logging config path {path} does not exist.")
-    with open(path, "rt") as f:
+    with open(path) as f:
```

Comment:
`"rt"` corresponds to the default `mode` for `open` (https://docs.astral.sh/ruff/rules/redundant-open-modes/#pyupgrade-up)